pkg/ingester/transfer.go

```diff
@@ -629,7 +629,10 @@ func unshippedBlocks(dir string) (map[string][]string, error) {
 
 		m, err := shipper.ReadMetaFile(filepath.Join(dir, userID))
 		if err != nil {
-			return nil, err
+			if !os.IsNotExist(err) {
```

Comment:
Could you comment explaining which is the case when it could happen and why we don't consider it an error?

Reply:
Added
```// If the cleanup function runs before the shipper has had a chance to run there wont be a meta file written yet.
// So we can treat as an "empty" meta file and no blocks have been shipped.```